homeassistant/components/roon/media_player.py

```diff
@@ -98,24 +98,19 @@ def __init__(self, server, player_data):
         """Initialize Roon device object."""
         self._remove_signal_status = None
         self._server = server
-        self._available = True
-        self._last_position_update = None
+        self._attr_available = True
         self._supports_standby = False
-        self._state = MediaPlayerState.IDLE
-        self._unique_id = None
+        self._attr_state = MediaPlayerState.IDLE
         self._zone_id = None
         self._output_id = None
-        self._name = DEVICE_DEFAULT_NAME
-        self._media_title = None
-        self._media_album_name = None
-        self._media_artist = None
```

Comment:
ditto

Reply:
Replaced with `_attr_media_artist`. The default value is `None` in the base class.